packages/agents-copilotstudio-client/src/copilotStudioClient.ts

```diff
@@ -89,7 +95,8 @@ export class CopilotStudioClient {
       url: uriStart,
       headers: {
         Accept: 'text/event-stream',
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
+        'User-Agent': CopilotStudioClient.getProductInfo(),
```

Comment:
This line,  + the call getProductInfo seems like its going to yield : 

"CopilotStudioClient.CopilotStudioClient.agents-sdk-js/version .... " 

For this I think we need : 

CopilotStudioClient-js/version ... 

thoughts?